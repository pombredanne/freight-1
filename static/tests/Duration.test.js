import React from 'react';
import renderer from 'react-test-renderer';
import Duration from '../components/Duration.jsx';

test('Duration Snapshot', () => {
  const component = renderer.create(
    (<Duration seconds="13.67" className="duration" />)
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
