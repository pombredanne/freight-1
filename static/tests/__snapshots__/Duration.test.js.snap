// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Duration Snapshot 1`] = `
<span
  className="duration"
>
  14 sec
</span>
`;
